stdlib/public/core/Sort.swift

```diff
@@ -438,9 +438,6 @@ internal func _merge<Element>(
       }
     }
   }
-
-  // FIXME: Remove this, it works around rdar://problem/45044610
-  return true
```

Comment:
If we find the workaround is no longer needed, maybe the changes to `Sort.swift` should be a separate PR!

Reply:
Yes sure! That make senes.